assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss

```diff
@@ -22,4 +22,212 @@
   .component-left-margin-section__content {
     overflow-x: hidden;
   }
+
+  // ----- Product Block
+
+  .product-block {
+    .component-left-margin-section__content {
+      position: relative;
+
+      @include mq($from: tablet) {
+        border-left: 1px solid gu-colour(garnett-neutral-4);
+      }
+    }
+
+    .component-grid-image {
+      width: 100%;
+      display: block;
+
+      @include mq($from: mobileLandscape, $until: phablet) {
+        width: 460px;
+        margin: 0 auto;
+      }
+
+      @include mq($from: tablet) {
+        width: 345px;
+      }
+    }
+
+    .component-price-cta {
+      @include mq($until: desktop) {
+        margin-top: $gu-v-spacing * 2;
+      }
+    }
+
+    .component-price-cta__cancel {
+      background-color: #fff;
+    }
+  }
+
+  .product-block__product {
+    @include mq($from: phablet) {
+      display: inline-block;
+      width: 50%;
+      vertical-align: top;
+    }
+
+    @include mq($from: desktop) {
+      width: 485px;
+    }
+
+    @include mq($from: leftCol) {
+      width: 520px;
+    }
+  }
+
+  .product-block__image {
+    border-bottom: 1px solid gu-colour(garnett-neutral-4);
+    padding: 0 ($gu-h-spacing / 2);
+    margin-bottom: $gu-v-spacing / 2;
+
+    @include mq($from: desktop) {
+      position: relative;
+    }
+
+    .svg-penny-farthing-circles {
+      display: none;
+
+      @include mq($from: desktop) {
+        display: block;
+        position: absolute;
+        width: 250px;
+        bottom: 0;
+        left: 255px;
+      }
+
+      @include mq($from: leftCol) {
+        left: 220px;        
+      }
+    }
+  }
+
+  .product-block__heading {
+    font-family: $gu-egyptian-web;
+    font-weight: 200;
+    font-size: 22px;
+    line-height: 26px;
+    padding: 0 ($gu-h-spacing / 2) $gu-v-spacing;
+
+    @include mq($from: phablet) {
+      font-size: 28px;
+      line-height: 1.14;
+      width: 700px;
+    }
+
+    @include mq($from: desktop) {
+      font-size: 36px;
+      line-height: 1.11;
+      width: 600px;
+      padding-bottom: $gu-v-spacing * 3;
+    }
+  }
+
+  .product-block__copy {
+    padding: 0 ($gu-h-spacing / 2) $gu-v-spacing;
+  }
+
+  .product-block__product-heading {
+    color: #fff;
+    font-weight: bold;
+    font-family: $gu-egyptian-web;
+    line-height: 1;
+    display: inline;
+    padding: 0 5px 3px;
+  }
+
+  .product-block__product-description {
+    margin: $gu-v-spacing / 2 0 $gu-v-spacing;
+    font-size: 16px;
+    line-height: 1.25;
+    font-family: $gu-egyptian-web;
+  }
+
+  .component-feature-list {
+    font-size: 16px;
+    line-height: 1.25;
+    font-family: $gu-egyptian-web;
+    list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 10 10"><circle cx="5" cy="5" r="5" fill="#ff4e36"/></svg>');
```

Comment:
Might be better to break this out into a separate svg file?

Reply:
As discussed offline, once built this produces a [`data URI`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs).